### PR TITLE
[FIX] mail: add get modelParam in ActivityController

### PR DESCRIPTION
**Pull request by pyduf**
We prepare the following fix by adding a modelParam getter that can be overridden by sub-classes.

How to reproduce:
1. Schedule an activity for a document.
2. Go to activity view.
3. Select the document to preview in the inspector.
4. Rename the document.
5. Refresh the page.

The document name is not renamed.

In the kanban and list view, the document is updated because multiEdit model parameter is enabled. It is not the case for the activity view.

Similarly to what is done in list and kanban controller, we add a modelParam getter to the activity controller to allow to override the parameters in subclasses which will allows to enable the multiEdit parameter in the document activitty controller.

Task-3817795

### CODE REVIEW

File: addons/mail/static/src/views/activity/activity_controller.js [hunk 1]
```diff
@@ -42,6 +35,16 @@ export class ActivityController extends Component {
...
         this.activity = useService("mail.activity");
     }
+    
+    get modelParams() {
+        return {
+            activeFields: this.props.archInfo.activeFields,
+            resModel: this.props.resModel,
+            fields: this.props.fields,
+            viewMode: "activity",
+            rootState: this.props.state || {},
```
Comment (amdi-odoo) [inline addons/mail/static/src/views/activity/activity_controller.js:45]: It should probably be: `this.props.state?.rootState` :thinking: 

or simply set this line in the getter:
`const { rootState } = this.props.state || {};`
then
```
return {
  ...
  rootState,
}
```

**Review by amdi-odoo — COMMENTED** [review]
Hi @pyduf :smile: 
Just one comment :slightly_smiling_face: 
Also could you rephrase the commit name so that it stays on one line? It's just a bit better :smile: 
Thanks and cheers!

**Review by pyduf — COMMENTED** [review]
Hello @amdi-odoo , 

Thanks for the review and having pointed out an error. I have updated the code and answered your comment.

**Comment by std-odoo** [comment]
@pyduf Hi :)

I think the enterprise link in the commit message is not needed (it will be added automatically)

Otherwise LGTM :)

**Comment by std-odoo** [comment]
@robodoo r+

**Comment by pyduf** [comment]
Hello @std-odoo , 

There was a conflict (double import of "clickWait" function in the test). Now it is solved and the runbot is green. 
Could you re-validate it ?

Thanks

**Comment by std-odoo** [comment]
@robodoo r+
